src/frontend/src/handler/create_database.rs

```diff
@@ -30,6 +30,8 @@ pub async fn handle_create_database(
     if_not_exist: bool,
     owner: Option<ObjectName>,
     resource_group: Option<SetVariableValue>,
+    barrier_interval_ms: Option<u32>,
```

Comment:
Shall we have a struct `DatabaseParam` with these two fields, so that we can pass the two fields together? Seems they always appear together in the path of create database.